packages/client/hmi-client/src/components/widgets/tera-input-text.vue

```diff
@@ -83,8 +84,13 @@ function formatValue(value: string | number | undefined) {
 
 const updateValue = (event: Event) => {
 	const target = event.target as HTMLInputElement;
-	const value = target.value;
-	emit('update:model-value', value);
+	if (props.charactersToReject && !isEmpty(props.charactersToReject)) {
```

Comment:
Probably sufficient with just the `isEmpty` check, don't think you need to check the variable is falsy

Reply:
Yeah I tried that but typescript isn't aware of lodash checking for that so it still underlines the variables thinking I'd be passing a potentially undefined value